taxi/urls.py

```diff
@@ -1,14 +1,26 @@
 from django.urls import path
 
-from .views import index, CarListView, CarDetailView, DriverListView, DriverDetailView, ManufacturerListView
+from .views import index, CarListView, CarDetailView, DriverListView, DriverDetailView,\
+    ManufacturerListView, DriverCreateView, DriverDeleteView, ManufacturerCreateView,\
+    ManufacturerUpdateView, ManufacturerDeleteView, CarCreateView, CarUpdateView,\
+    CarDeleteView, DriverUpdateView
 
 urlpatterns = [
     path("", index, name="index"),
     path("manufacturers/", ManufacturerListView.as_view(), name="manufacturer-list"),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
-    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail")
+    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path("drivers/<int:pk>/delete/", DriverDeleteView.as_view(), name="driver-delete"),
+    path("drivers/<int:pk>/update/", DriverUpdateView.as_view(), name="driver-update"),
+    path("manufacturers/create/", ManufacturerCreateView.as_view(), name="manufacturer-create"),
+    path("manufacturers/<int:pk>/update/", ManufacturerUpdateView.as_view(), name="manufacturer-update"),
+    path("manufacturers/<int:pk>/delete/", ManufacturerDeleteView.as_view(), name="manufacturer-delete"),
+    path("create/", CarCreateView.as_view(), name="car-create"),
```

Comment:
this way isn't informative.
why do u use path `"manufacturers/create/"` for manufacturer but for car, do u use` "create/"`  why don't `"car/create/"`?